doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -415,90 +425,86 @@ For instance, if you have one cluster in North America and another in the EU, yo
+### Deploy the toystore app
 
+Create the namespace for your application. 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${devNS}
-```
 
+kubectl create ns ${DEVELOPER_NS}
+```
 
-### Step 2 - Set up HTTPRoute and backend
-
-Copy at least one of the following example OAS to a local location:
-
-- [Sample OAS for rate limiting with API key](../../examples/oas-apikey.yaml)
-
-- [Sample OAS for rate limiting with OIDC](../../examples/oas-oidc.yaml)
-
-Set up some new environment variables as follows:
+Deploy a new version of the toystore app to your developer namespace as follows: 
 
 ```bash
-export oasPath=examples/oas-apikey.yaml
-# Ensure you still have these environment variables setup from the start of this guide:
-export rootDomain=example.com
-export gatewayNS=api-gateway
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${DEVELOPER_NS}
 ```
 
-### Step 3 - Use OAS to define your HTTPRoute rules
-
-You can generate Kuadrant and Gateway API resources directly from OAS documents by using an `x-kuadrant` extension.
-
-NOTE: For a more in-depth look at the OAS extension, see the [kuadrantctl documentation](https://docs.kuadrant.io/latest/kuadrantctl/).
-
-You will use `kuadrantctl` to generate your `HTTPRoute`.
-
-NOTE: The sample OAS has some placeholders for namespaces and domains. You will inject valid values into these placeholders based on your previous environment variables.
-
-Generate the resource from your OAS as follows, (`envsubst` will replace the placeholders):
+### Set up HTTPRoute and backend
 
+Create a HTTPRoute
 ```bash
-cat $oasPath | envsubst | kuadrantctl generate gatewayapi httproute --oas - | kubectl apply -f -
+
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: toystore
+  labels:
+    deployment: toystore
+    service: toystore
+spec:
+  parentRefs:
+  - name: ${gatewayName}
+    namespace: ${gatewayNS}
+  hostnames:
+  - "api.${rootDomain}"
+  rules:
+  - matches:
+    - method: GET
+      path:
+        type: PathPrefix
+        value: "/cars"
+    - method: GET
+      path:
+        type: PathPrefix
+        value: "/dolls"
+    backendRefs:
+    - name: toystore
+      port: 80
+  - matches:
+    - path:
+        type: PathPrefix
+        value: "/admin"
+    backendRefs:
+    - name: toystore
+      port: 80
+EOF
 ```
 
+You can access your API endpoint over HTTPS as follows: 
 ```bash
-kubectl get httproute toystore -n ${devNS} -o=yaml
-```
 
-You should see that this route is affected by the `AuthPolicy` and `RateLimitPolicy` defined as defaults on the Gateway in the Gateway namespace.
+export INGRESS_HOST=$(kubectl get gtw external${gatewayName} -o jsonpath='{.status.addresses[0].value}' -n api-gateway)
+
+curl --resolve api.${rootDomain}:443:${INGRESS_HOST} "https://api.${rootDomain}/cars"
```

Comment:
Not sure why we are doing it like this? I think we could do it like in the above example:

```
curl -k -w "%{http_code}" "https://$(kubectl get httproute test -n ${GATEWAY_NS} -o=jsonpath='{.spec.hostnames[0]}')"
```

Also could use the curl pod again